src/hcf-release/jobs/acceptance-tests-brain/templates/pre-start.erb

```diff
@@ -8,3 +8,4 @@ PATH=$PATH:/var/vcap/packages/cli/bin
 cf api --skip-ssl-validation "https://api.$DOMAIN"
 cf auth admin "${CLUSTER_ADMIN_PASSWORD}"
 cf enable-feature-flag diego_docker
+
```

Comment:
We don't really _need_ a blank line…
But we don't _need_ to _not_ have a blank line either.

Reply:
Sure, that is trivial. Fixed with commit ff78a6a167371665a4c4df88833e0c8cebc13597.